src/tiledb/cloud/soma/ingest.py

```diff
@@ -355,12 +230,97 @@ def __getattr__(self, name: str) -> object:
     logging.info("Successfully wrote data from %s to %s", input_uri, output_uri)
 
 
-# Until we fully get this version of tiledb.cloud deployed server-side, we must
```

Comment:
Removing this makes it impossible for me to locally test -- unless `as_batch` does this for me -- ?

@JohnMoutafis please confirm ... 🙏

Reply:
@johnkerl `as_batch` will run a function as a batch UDF on TileDB Cloud.
Have you tried running either `run_ingest_workflow` or `ingest` from your local?

Reply:
@JohnMoutafis I need to be able to edit a function and run it and have the remote be running my changed code, not the latest tagged version.

I haven't yet tried locally cloning this PR of yours as an experiment. I'll do so, before approving this PR, to verify that editable execution hasn't been broken by this PR.

Thanks!

Reply:
I did `pip install -v -e .`, edited locally (adding a print statement), and ran an ingest -- and confirmed that I did _not_ see my mods being picked up by the remote executor. (More details in Slack.)